server.js

```diff
@@ -1,25 +1,24 @@
-const liveServer = require('live-server')
-const isSSR = !!process.env.SSR
-const middleware = []
+const liveServer = require('live-server');
+const isSSR = !!process.env.SSR;
+const middleware = [];
 
 if (isSSR) {
-  const Renderer = require('./packages/docsify-server-renderer/build.js')
+  const { initJSDOM } = require('./test/_helper');
+
+  const dom = initJSDOM();
+  dom.reconfigure({ url: 'https://127.0.0.1:3000' });
+
+  require = require('esm')(module /* , options */);
+
+  const {
+    Renderer,
+    getDefaultTemplate,
+  } = require('./packages/docsify-server-renderer/index');
```

Comment:
Import the ES Modules. Much nicer experience in the debugger (Chrome devtools).

Try it:

```
npm i -g ndb
SSR=true ndb server.js
```